packages/eventsub-base/src/events/EventSubChannelUpdateEvent.ts

```diff
@@ -83,8 +83,24 @@ export class EventSubChannelUpdateEvent extends DataObject<EventSubChannelUpdate
 
 	/**
 	 * Whether the channel is flagged as suitable for mature audiences only.
+	 *
+	 * @deprecated Use {@link EventSubChannelUpdateEvent#contentClassificationLabels} to check if any content
+	 * classification labels are applied to the channel.
+	 *
+	 * Currently, this flag mimics the previous behavior by checking whether the `contentClassificationLabels`
+	 * array is not empty.
+	 *
+	 * This flag will be removed in the next major release.
```

Comment:
> This flag will be removed in the next major release

Where do you see this? https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types/#channelupdate

Reply:
@jumoog This is a reference to `channel.update` v2, while the library still uses v1.

You can find this entry in the [changelog](https://dev.twitch.tv/docs/change-log/):

2023‑06‑29 	

Moved the following changes to existing subscription type [channel.update](https://dev.twitch.tv/docs/eventsub/eventsub-subscription-types/#channelupdate) to open beta:

  - Removed the `is_mature` boolean flag
  - Added a new `content_classification_labels` field that lists the IDs of any [content classification labels](https://safety.twitch.tv/s/article/Content-Classification-Guidelines) applied on the channel

Reply:
Yes, I think this is a valid migration strategy to avoid a breaking change. It also makes sense to remove it as soon as possible (i.e. semver-major).